schema/mapem/mapem_schema_2-0-0.json

```diff
@@ -0,0 +1,691 @@
+                "$ref": "../dsrc/dsrc_schema_2-0-0.json#/definitions/speed_limit_list"
+              },
+              "lane_set": {
+                "description": "Data about one or more lanes (all lane data is found here) Data describing how to use and request preemption and priority services from this intersection (if supported).",
+                "$ref": "#/$defs/road_lane_set_list"
+              }
+            }
+          }
+        },
+        "road_segments": {
+          "type": "array",
+          "description": "List of RoadSegment entries.",
+          "minItems": 1,
+          "maxItems": 32,
+          "items": {
+            "type": "object",
+            "description": "Complete description of a RoadSegment including its geometry and its allowed navigational paths (independent of any additional regulatory restrictions that may apply over time or from user classification) and any current disruptions such as a work zone or incident event.",
+            "required": [
+              "id",
+              "revision",
+              "ref_point",
+              "road_lane_set"
+            ],
+            "properties": {
+              "name": {
+                "type": "string",
+                "description": "For debug use only."
+              },
+              "id": {
+                "$ref": "../dsrc/dsrc_schema_2-0-0.json#/definitions/road_segment_reference_id"
+              },
+              "revision": {
+                "type": "integer",
+                "minimum": 0,
+                "maximum": 127
+              },
+              "ref_point": {
+                "description": "The reference from which subsequent data points are offset until a new point is used.",
+                "$ref": "../dsrc/dsrc_schema_2-0-0.json#/definitions/position_3d"
+              },
+              "lane_width": {
+                "description": "Reference width used by all subsequent lanes unless a new width is given.",
+                "$ref": "../dsrc/dsrc_schema_2-0-0.json#/definitions/lane_width"
+              },
+              "speed_limits": {
+                "description": "Reference regulatory speed limits used by all subsequent lanes unless a new speed is given.",
+                "$ref": "../dsrc/dsrc_schema_2-0-0.json#/definitions/speed_limit_list"
+              },
+              "road_lane_set": {
+                "description": "Description of a roadway segment. Data describing disruptions in the RoadSegment such as work zones etc will be added here.",
+                "$ref": "#/$defs/road_lane_set_list"
+              }
+            }
+          }
+        },
+        "data_parameters": {
+          "type": "object",
+          "description": "Used to provide basic (static) information on how a map fragment was processed or determined.",
+          "properties": {
+            "process_method": {
+              "$ref": "../dsrc/dsrc_schema_2-0-0.json#/definitions/ia5_string"
```

Comment:
As discussed in the DDSRC schema: make those plain JSON strings.

Reply:
Done.